content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: L7 - waypoint proxy Layer
+description: User guide for L7 Processing.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+{{<tip>}}
+Before you start with this guide, please make sure that you have already read the Ztunnel Networking sub-guide and have a basic understanding of Istio Ambient Mesh. This guide assumes that you have already completed the Installation, set up the ztunnel and enabled mTLS.
+
+If you have not yet done these things, please go back to the Ztunnel Networking sub-guide and follow the instructions before proceeding with this guide.
```

Comment:
Could remove this line .. you are already stating above and below that ztunnel is a pre-requisite

Reply:
sure, done!

Reply:
pre-requisites would be nice, especially if links won't work in tip section.